ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -275,6 +275,75 @@ objects:
         /bin/openshift-install --dir=/tmp/artifacts/installer --log-level=debug create cluster &
         wait "$!"
 
+    # Stream "journal -u bootkube" and "kubectl get all -a --all-namespaces" into artifact
+    - name: stream-bootstrapping
+      image: ${IMAGE_INSTALLER}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /etc/openshift-installer
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        mkdir -p .ssh
+        chmod 700 .ssh
+        cp /etc/openshift-installer/ssh-privatekey .ssh/id_rsa
+        cp /etc/openshift-installer/ssh-publickey .ssh/id_rsa.pub
+
+        function stream-bootkube () {
+          ip=${1}
+          while true; do
+            ssh -o "StrictHostKeyChecking=no" core@${ip} sudo journal -u bootkube 2>&1 >> /tmp/artifacts/bootstrap/bootkube.log
```

Comment:
nit: can we use the fully-qualified `bootkube.service`?

Also, this redirect isn't quite right, since you redirect stderr into stdout before adjusting stdout.  For example:

```console
$ (echo hi >&2) 2>&1 >>/tmp/bootkube.log
hi
```

You should instead redirect stdout first, and then redirect stderr into stdout:

```console
$ (echo hi >&2) >>/tmp/bootkube.log 2>&1
$ cat /tmp/bootkube.log 
hi
```